Resources/Prototypes/_RMC14/Species/vulpkanin.yml

```diff
@@ -75,9 +75,6 @@
       points: 1
       required: true
       defaultMarkings: [ VulpEar ]
-    Overlay:
-      points: 2
-      required: false
```

Comment:
Was removing this intended?

Reply:
Yes, they have 2 existing overlay markings that would set a limit on the amount of bandage markings you could use

no other species (as far as I could tell, anyway) had a limit on overlay markings, so I removed that field entirely to prevent there being a limit on the amount of gauze/bandage markings you can apply